### ISSUE TITLE
Clarify material design dependency

### ISSUE THREAD

**Issue by liedQM** [AltBeacon/android-beacon-library#1048]
### Expected behavior
A library without UI should not depend on a UI framework like the material design library.
https://material.io/develop/android/docs/getting-started

### Actual behavior
The library depends on `'com.google.android.material:material:1.3.0'`.

### Android Beacon Library version 
`2.17.1` didn't have a material design dependency
Starting from `2.18` the library depends on material design.

We are using your beacon scanner library in one of our libraries which also have no UI and the change enforces our library to also depend on an UI framework which actually makes no sense in my opinion.

I like to ask here:
Why does the library depend on the material design library without having any UI component or any mention of the design library in the code?

**Comment by liedQM** [comment]
I can't even find any usage of `com.google.android.material` in your repository:
[com.google.android.material&type=code](https://github.com/AltBeacon/android-beacon-library/search?q=com.google.android.material&type=code)

**Comment by davidgyoung** [comment]
What makes you say “The library depends on 'com.google.android.material:material:1.3.0'”?  There is certainly no intention that this be true.

**Comment by liedQM** [comment]
Your pom tells me otherwise:
https://repo1.maven.org/maven2/org/altbeacon/android-beacon-library/2.19/android-beacon-library-2.19.pom

```<dependency>
<groupId>com.google.android.material</groupId>
<artifactId>material</artifactId>
<version>1.3.0</version>
</dependency>```

**Comment by davidgyoung** [comment]
Fixed in #1049
